packages/manager/cypress/e2e/core/linodes/linode-network.spec.ts

```diff
@@ -0,0 +1,89 @@
+import type { Linode } from '@linode/api-v4';
+
+import { createLinodeRequestFactory } from '@src/factories';
+
+import { authenticate } from 'support/api/authentication';
+import { interceptGetLinode } from 'support/intercepts/linodes';
+import { interceptUpdateIPAddress } from 'support/intercepts/networking';
+import { cleanUp } from 'support/util/cleanup';
+import { createTestLinode } from 'support/util/linodes';
+import { randomLabel } from 'support/util/random';
+import { chooseRegion } from 'support/util/regions';
+import { ui } from 'support/ui';
+
+authenticate();
+describe('linode networking', () => {
+  before(() => {
+    cleanUp('linodes');
+  });
+
+  /**
+   * - Confirms flow for editing the RDNS of an IP address
+   * - Confirms toast message upon editing RDNS
+   */
+  it('can edit the RDNS of an IP address', () => {
+    const createLinodeRequest = createLinodeRequestFactory.build({
+      label: randomLabel(),
+      region: chooseRegion().id,
+      backups_enabled: false,
+      booted: false,
+    });
+
+    cy.defer(
+      () => createTestLinode(createLinodeRequest),
+      'creating Linode'
+    ).then((linode: Linode) => {
```

Comment:
```suggestion
    cy.defer(
      () => createTestLinode(null, { securityMethod: 'powered_off' }),
      'creating Linode'
    ).then((linode: Linode) => {
```

You can save a few lines here by using the [default payload](https://github.com/linode/manager/blob/a678228/packages/manager/cypress/support/util/linodes.ts#L108-L113)! Since we want a powered off Linode anyway, you can avoid having a Firewall created/attached by passing the `securityMethod: 'powered_off'` option (which will enforce the Linode to be created with `booted: false` regardless of whatever is set in the payload).